docker/all-in-one/entrypoint.sh

```diff
@@ -138,10 +175,28 @@ fi
 
 # Download and extract init payload from s3
 export INIT_PAYLOAD_PATH=${INIT_PAYLOAD_PATH:-/tmp/payload.tar.gz}
-export CONFIGURED_FLAG_PATH=${CONFIGURED_FLAG_PATH:-$PGDATA/../machine.configured}
 
 if [ "${INIT_PAYLOAD_PRESIGNED_URL:-}" ]; then
-  curl -sSL "$INIT_PAYLOAD_PRESIGNED_URL" -o "$INIT_PAYLOAD_PATH"
+  curl -sSL "$INIT_PAYLOAD_PRESIGNED_URL" -o "/tmp/payload.tar.gz"
+  mv "/tmp/payload.tar.gz" "$INIT_PAYLOAD_PATH"
+fi
+
+if [ "${DATA_VOLUME_MOUNTPOINT}" ]; then
+  LOGS_FOLDER="${DATA_VOLUME_MOUNTPOINT}/logs"
+
+  mkdir -p "${LOGS_FOLDER}/postgresql"
```

Comment:
could be a loop over postgresql, services, wal-g, but this is fine too

Reply:
fair point - refactored into a loop